### ISSUE TITLE
test image folder can not be reused

### ISSUE THREAD

**Issue by hannraoi** [leehomyc/Photo-Realistic-Super-Resoluton#13]
Hi~it's me again and this time I have met another problem =)

The first time I run "t_folder=/xx/xx model_file=/xx/xx result_path=/xx/xx th run_test.lua" and it worked perfectly, which is nice, but after I change the test images under the same folder, it reminds me that a certain image file can not be found, the error info is like this:

hannraoi@hannraoi-B85M-DS3H:~/Photo-Realistic-Super-Resoluton-master$ t_folder=/home/hannraoi/my_srgan_test/ model_file=/home/hannraoi/my_srgan_model/super_resolution_adversarial_G_134 result_path=/home/hannraoi/my_srgan_result/ th run_test.lua 
{
  niter : 250
  dataset : "folder"
  ntrain : inf
  model_file : "/home/hannraoi/my_srgan_model/super_resolution_adversarial_G_134"
  t_folder : "/home/hannraoi/my_srgan_test/"
  batchSize : 32
  result_path : "/home/hannraoi/my_srgan_result/"
  loadSize : 96
  nThreads : 4
  scale : 4
  gpu : 1
}
Starting donkey with id: 1 seed: 1
table: 0x7fd2180ff0a0
Starting donkey with id: 3 seed: 3
table: 0x7fd214129350
Starting donkey with id: 2 seed: 2
table: 0x7fd210128ad0
Starting donkey with id: 4 seed: 4
table: 0x7fd20c128da0
Loading train metadata from cache
Loading train metadata from cache
Loading train metadata from cache
Loading train metadata from cache
/home/hannraoi/torch/install/bin/lua: ...hannraoi/torch/install/share/lua/5.2/threads/threads.lua:183: [thread 2 callback] /home/hannraoi/torch/install/share/lua/5.2/image/init.lua:352: /home/hannraoi/my_srgan_test/subfolder/1b38b72e-af9c-4708-8e7d-5ba80d4a24ea.png: No such file or directory
stack traceback:
	[C]: in function 'error'
	/home/hannraoi/torch/install/share/lua/5.2/image/init.lua:352: in function 'load'
	...tic-Super-Resoluton-master/data/donkey_folder_supres.lua:35: in function 'loadImage'
	...tic-Super-Resoluton-master/data/donkey_folder_supres.lua:50: in function <...tic-Super-Resoluton-master/data/donkey_folder_supres.lua:48>
	(...tail calls...)
	.../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:354: in function <.../Photo-Realistic-Super-Resoluton-master/data/dataset.lua:347>
	(...tail calls...)
	[C]: in function 'xpcall'
	...hannraoi/torch/install/share/lua/5.2/threads/threads.lua:234: in function 'callback'
	...e/hannraoi/torch/install/share/lua/5.2/threads/queue.lua:65: in function <...e/hannraoi/torch/install/share/lua/5.2/threads/queue.lua:41>
	[C]: in function 'pcall'
	...e/hannraoi/torch/install/share/lua/5.2/threads/queue.lua:40: in function 'dojob'
	[string "  local Queue = require 'threads.queue'..."]:15: in main chunk
stack traceback:
	[C]: in function 'error'
	...hannraoi/torch/install/share/lua/5.2/threads/threads.lua:183: in function 'dojob'
	...aoi/Photo-Realistic-Super-Resoluton-master/data/data.lua:76: in function 'getBatch'
	run_test.lua:33: in main chunk
	[C]: in function 'dofile'
	...raoi/torch/install/lib/luarocks/rocks/trepl/scm-1/bin/th:150: in main chunk
	[C]: in ?

I know little about Lua, so I don't know where the problem comes from, I hope to get some help, thanks!

**Comment by hannraoi** [comment]
by the way, my solution to the problem is to create a new t_folder with a different name each time I run the test code.

**Comment by smahajan07** [comment]
Hi @hannraoi . How many images did you use from Image net to train the model? And which data did you test your model on?

I have trained on a data of faces (approx 200,000) till 50 epochs, but the results are not great.

**Comment by hannraoi** [comment]
hi~ @smahajan07 
I'm sorry that I didn't use ImageNet data as my training data, I use my own data instead and the amount of data is a little less than urs.
